### ISSUE TITLE
  The latest version of Firefox (58.0.x...

### ISSUE THREAD

**Issue by wormbasehelp** [WormBase/website#6202]
*Submitted from the feedback form on the WormBase website.*

<p>&nbsp;&nbsp;The latest version of Firefox (58.0.x  64-Bit) does not show any wormbase content apart from the navigation. Clicking links in the navigation parts has no effect. This applies to the Mac as well as the Windows version.</p>

**Reported by:** mark******** (mark***********************)
**Submitted from:** <a target="_blank" href="http://www.wormbase.org//tools/support">/tools/support</a>
**Browser:** Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/604.5.6 (KHTML, like Gecko) Version/11.0.3 Safari/604.5.6

**Comment by hansduo** [comment]
Hold on - my fault. Jquery was blocked by some third party extensions. This issue can be closed, I guess.

**Comment by chris-grove** [comment]
Hi Mark, apologies for the bug. Our web team has implemented a potential fix. Is this still an issue for you?


If it is still an issue, could you check if the URL you are looking at is http or https? We've noticed problems like the one you're describing when viewing WormBase from the https URL.


Best,


Chris Grove
WormBase

**Comment by hansduo** [comment]
You are right. Although I CAN make the content re-appear by disabling script blocking add-ons, the content will still not show up if I use https.

**Comment by raymond91125** [comment]
duplicate of #6135?

**Comment by nuin** [comment]
The problem only shows on `https` access, regular `http` seems fine. With `https` set, bot Safari and Firefox have the same behaviour and the errors is on loading `jquery` libraries with `http` only, as the browsers are locking based on mixed protocols

```
Blocked loading mixed active content “http://code.jquery.com/ui/1.10.1/jquery-ui.min.js”[Learn More]
www.wormbase.org
Blocked loading mixed active content “http://code.jquery.com/ui/1.10.1/jquery-ui.min.js”[Learn More]```
